rasa/utils/common.py

```diff
@@ -66,6 +66,9 @@
     # is not available on PyPi, so we cannot pin the newer version.
     # cf. https://github.com/google/flatbuffers/issues/6957
     (DeprecationWarning, "the imp module is deprecated in favour of importlib.*"),
+    # Cannot fix this deprecation warning since we need to support two
+    # numpy versions as long as we keep python 37 around
+    (DeprecationWarning, "the `interpolation=` argument to quantile was renamed"),
```

Comment:
nitpicking but we could just add this warning `if sys.version_info.minor == 7`

Reply:
we'd have to do it for`!=7` because that's where the deprecation warning will come. 7 still uses the old version which won't have a warning. I'll think about it.